spring-pulsar-docs/src/main/asciidoc/quick-tour.adoc

```diff
@@ -1,6 +1,8 @@
 [[quick-tour]]
 === Quick Tour
 
+:spring-pulsar-version: 0.1.0-SNAPSHOT
```

Comment:
An improvement will be to figure out how to populate this so as to not have another place to update in the code when we change versions.